src/posthog-surveys-types.ts

```diff
@@ -107,3 +109,39 @@ export interface Survey {
     start_date: string | null
     end_date: string | null
 }
+
+// Use the Fisher-yates algorithm to shuffle this array
+// https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle
+const shuffle = (array: any[]) => {
```

Comment:
You can put these two functions in `survey-utils` - we generally don't want implementation code in the types file - this is solely for expressing common types.

Reply:
This is fixed now.